### ISSUE TITLE
Add ability to include additional instances' feed in home timeline (or filter federated timeline by instance)

### ISSUE THREAD

**Issue by hypertelia** [mastodon/mastodon#20995]
### Pitch

Users may have a variety of interests that are served by topic-specific instances. Choosing which of these to join is a dilemma. If users could add a secondary (or more) feed to their home timeline they'd be able to follow activity from these instances alongside their local posts. 

Alternatively being able to filter which instances appear in a federated timeline could serve the same purpose.

### Motivation

This would benefit those using a general instance with interests that are also served by topic-focussed instances. Following a hashtag doesn't quite cut it, as users on a topic-focussed instance are less likely to use essentially a self-describing hashtag. ie. it's always say, #cooking as this is @cooking.social.

**Comment by Timothyjchambers** [comment]
Should we merge this and #20379? and #21327?  I agree it is a very important new feature that would help users in discovery of new content in big ways.

**Comment by cr08** [comment]
> Should we merge this and #20379? and #21327? I agree it is a very important new feature that would help users in discovery of new content in big ways.

I definitely think an overarching discussion in better content discoverability with Mastodon is worthwhile and whatever ideas under that umbrella people recommend. Reading through some of the issues I can definitely see a lot of them being good ideas but finding one that works for everyone is going to be the big discussion to have here I think.

**Comment by yllekz** [comment]
Yes. Absolutely need the ability to follow instances in addition to individual people. I feel like I'm missing entire conversations on certain instances due to the lack of this feature.
